Ball of Duty Server/Domain/Maps/Map.cs

```diff
@@ -243,13 +248,9 @@ public void Update(Observable observable, object data)
         /// <param name="e"></param>
         public void DecayScores(object sender, ElapsedEventArgs e)
         {
-            foreach (var go in GameObjects.Values)
+            foreach (Character character in GetCharacters())
```

Comment:
we should refactor this. move the responsibility to the character itself. that is what the new overridable update method is for